models/stg_recharge__address.sql

```diff
@@ -32,8 +32,10 @@ final as (
         province,
         zip,
         country_code,
+        country,
         company,
-        phone
+        phone,
+        payment_method_id
```

Comment:
We technically will want to make this a breaking change and call it out in the CHANGELOG. These fields _could_ have been used in the passthrough column variable and could result in a breaking run if they are still included.